### DISCUSSION TITLE
Sorted Tablist

### DISCUSSION THREAD

**Discussion by Insprill** [Zrips/CMI#2671] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Basically the ability to sort tablist according to Rank (with vault support) So like say owner then mvp++ then mvp + etc
### Basic explanation as to why this feature is needed and/or useful to have: ###
So people can see the higher ranks first and so like admins don't get lost in a sea of people

**Comment by digishield** [comment]
Yes please! Sorting a TAB is one of the main things that keeps me looking for another plugin to replace CMI TAB. And I don't want unnecessary plugins when CMI can do everything I need. But random TAB sorting drives me crazy :)

**Comment by Insprill** [comment]
Honesty, TAB [1.5.x - [1.15.2] REBORN is better. It offers alot more features then CMI Tablist does. And it can set nameplate for players aswell. Updates almost instantly with luckperms if u change a users parent

**Comment by digishield** [comment]
Yes, my fault. I expressed myself in a wrong way. I meant CMI Tablist of course. I used TAB Reborn all the time, but then I didn't want to have two plugins that do the same thing. And especially TAB Reborn does not support CMI placeholders on AFK symbol, etc. Therefore, I would like CMI Tablist to do the same as TAB Reborn or at least be able to sort players according to what I want.

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
